Resources/Prototypes/Entities/Objects/Consumable/drinks_cans.yml

```diff
@@ -54,6 +54,7 @@
   - type: Tag
     tags:
     - Cola
+    - Trash
```

Comment:
maybe give this to DrinkCanBaseFull/Empty?

Reply:
It has it already, having a new tag component in the child prototype removed the parent tag. Currently in game you can't trash bag a cola can, you can everything else.